src/config.ts

```diff
@@ -1,5 +1,31 @@
 import { Args } from "grimoire-kolmafia";
-import { get } from "libram";
+import { Item } from "kolmafia";
+import { $item, $items, get } from "libram";
+
+const workshedAliases = [
+  { item: $item`cold medicine cabinet`, aliases: ["cmc"] },
+  { item: $item`model train set`, aliases: ["trainset", "trainrealm", "train"] },
+  { item: $item`Asdon Martin keyfob`, aliases: ["breadcar", "car", "asdon", "aston"] },
+  { item: $item`diabolic pizza cube`, aliases: ["cube", "pizza", "pizzacube"] },
+  { item: $item`portable Mayo Clinic`, aliases: ["mayo", "clinic"] },
+  { item: $item`Little Geneticist DNA-Splicing Lab`, aliases: ["dna", "lab", "dnalab"] },
```

Comment:
Like Katarn has also mentioned, I don't really like `lab` since the chemistry lab is also a lab. This also complicates matters if we do intend to reintroduce substring matching in the future, where `lab` (which may be a working implementation in some user's scripts) would then cause issues with the reimplementation of that feature.